hello_world.py

```diff
@@ -1,12 +1,44 @@
-#!/bin/bash
+#!/usr/bin/env python
 
 import argparse
 
+
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("name")
     return parser.parse_args()
 
+
+def is_name_significant(name):
```

Comment:
`is_name_significant` is an ok name I guess but I generally see that pattern with functions that return booleans. On the other hand it could also be an acceptable function in this case but maybe camel case would make it more apparent as to what this is doing. Try following the PEP8 style guide but don't follow it to the letter because you know one of the rules is to use it as a guide and not a hard rulebook lol